plugin/src/main/java/com/denizenscript/denizen/objects/PlayerTag.java

```diff
@@ -2339,6 +2339,17 @@ else if (foodLevel / maxHunger < 1) {
             return null;
         });
 
+        // <--[tag]
+        // @attribute <PlayerTag.skin_model>
+        // @returns ElementTag
+        // @description
+        // Returns the player's skin model, either CLASSIC or SLIM.
+        // Requires the player to be online.

```

Comment:
This line shouldn't be in meta, it's implied for all tags not explicitly documented as working with offline players